modules/ensemble/lib/action/dialog_actions.dart

```diff
@@ -18,11 +18,13 @@ class ShowDialogAction extends EnsembleAction {
   ShowDialogAction({
     super.initiator,
     required this.body,
+    this.dismissible = true,
```

Comment:
don't hardcode the default here, you already have a default when reading the payload. Bad design to hardcode in multiple places

Reply:
removed hard coded default value